content/docs/04.6-vm-management/15-vm-roles-permissions.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "VM User Roles and Permissions"
+metaTitle: "VM User Roles and Permissions"
+metaDescription: "Learn how to "
+icon: " "
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Overview
+
+You must configure roles and role binding before any user, including you as administrator, can access the VM Management functionality in Palette. There are two sets of roles: Cluster Roles and Palette Roles, along with the required bindings configuration.
+
+Palette provides the following four out-of-the-box Cluster roles for VM Management. The table also lists the corresponding Palette roles. 
+
+<br />
+
+| Cluster Role  | Description | Restrictions | Palette Role |
```

Comment:
@ritawatson The palette role name are incorrect. It should be Virtual Machine Admin instead of VM Admin
![image](https://user-images.githubusercontent.com/1317641/226262264-f34c4989-129b-4b7e-95b4-14ab929f5528.png)

Reply:
spelled out Virtual Machine in each instance.